tests/test_device_mesh.py

```diff
@@ -11,27 +11,17 @@
 import numpy as np
 import ray
 
-from alpa import DeviceCluster, parallelize, set_parallelize_options, testing, DistributedArray
+from alpa import init, parallelize, DistributedArray
+from alpa.device_mesh import get_global_physical_mesh
 from alpa.testing import assert_allclose
 
 
 class DeviceMeshTest(unittest.TestCase):
 
     def setUp(self):
-        os.environ["XLA_PYTHON_CLIENT_ALLOCATOR"] = "platform"
-        ray.init(address="auto", ignore_reinit_error=True)
-
-    def tearDown(self):
```

Comment:
Why don't we need to shut down ray anymore?

Reply:
Yes, because we support multiple computational graphs